core/api/src/main/java/org/trellisldp/api/SpecialResources.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.api;
+
+import java.time.Instant;
+import java.util.stream.Stream;
+
+import org.apache.commons.rdf.api.IRI;
+import org.apache.commons.rdf.api.Quad;
+
+/**
+ * Singleton "special" resources.
+ */
+public enum SpecialResources implements Resource {
```

Comment:
Should this be a static inner type of `Resource`?

Reply:
I could go either way on this, but if you'd prefer it to be located in the `Resource` interface, I'll move it.

Reply:
Yeah, so that it's that much harder to miss them, I would appreciate that.